### PR TITLE
Ima 20

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
##what
setup ima service in hof-rds-api

##Why
new ima service

##How
Added ima service with migration ,seeds, and db configs

### CODE REVIEW

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "start:dev": "nodemon -r dotenv/config --ignore data/ server.js",
     "db:setup": "yarn db:migrate && yarn db:seed",
     "db:test": "yarn db:setup && yarn run db:rollback",
-    "db:migrate": "node -e 'require(\"./db\").migrate()'",
+    "db:migrate": "node -r dotenv/config -e 'require(\"./db\").migrate()'",
```
[the remark this answers is not in the document]
Comment (TemitopeAyokuHO) [reply package.json]: Done

File: services/ima/migrations/20230928142449_create_saved_applications.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+exports.up = function(knex) {
+    return knex.schema.createTable('saved_applications', table => {
+        table.increments();
+        table.integer('caseworker_id').unsigned().notNullable().references('id').inTable('caseworkers');
+        table.string('uan').notNullable();
+        table.string('email');
+        table.string('date_of_birth').notNullable();
+        table.json('session').notNullable();
+        table.timestamps(true, true);
+        table.string('submitted_at');
```
[the remark this answers is not in the document]
Comment (TemitopeAyokuHO) [reply services/ima/migrations/20230928142449_create_saved_applications.js]: Done

**Review by Rhodine-orleans-lindsay — APPROVED** [review]
lgtm

**Review by Rhodine-orleans-lindsay — COMMENTED** [review]
can you squash the 2 commits please?

File: services/ima/seeds/01_saved_applications.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      return knex('saved_applications').insert([
+        {
+          email: 'temitope.ayoku@digital.homeoffice.gov.uk',
+          uan: '1234-5645-8670-0000',
+          date_of_birth: '2000/01/01',
+          session: '{}',
+          submitted_at: '2023/09/09'
+        },
+        {
+          email: 'temitope.ayoku@digital.homeoffice.gov.uk',
```
Comment (Rhodine-orleans-lindsay) [inline services/ima/seeds/01_saved_applications.js:19]: I would use a test email

File: services/ima/seeds/01_saved_applications.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+// eslint-disable-next-line func-names
+exports.seed = function (knex) {
+  // Deletes ALL existing entries
+  return knex('saved_applications').del()
+    // eslint-disable-next-line func-names
+    .then(function () {
+      // Inserts seed entries
+      return knex('saved_applications').insert([
+        {
+          email: 'temitope.ayoku@digital.homeoffice.gov.uk',
```
Comment (Rhodine-orleans-lindsay) [inline services/ima/seeds/01_saved_applications.js:12]: I would use a test email

**Pull request merged**